src/main/java/frc/robot/swerve/SwerveDrive.java

```diff
@@ -17,26 +16,29 @@ public class SwerveDrive extends SubsystemBase {
     private final SwerveModule[] modules = new SwerveModule[4]; //FL, FR, RL, RR
     private final LinearFilter accelFilter = LinearFilter.movingAverage(15);
     private final SwerveDriveKinematics kinematics = new SwerveDriveKinematics(
-            SwerveConstants.wheelPositions[0],
-            SwerveConstants.wheelPositions[1],
-            SwerveConstants.wheelPositions[2],
-            SwerveConstants.wheelPositions[3]
+            SwerveConstants.WHEEL_POSITIONS[0],
+            SwerveConstants.WHEEL_POSITIONS[1],
+            SwerveConstants.WHEEL_POSITIONS[2],
+            SwerveConstants.WHEEL_POSITIONS[3]
     );
     private final SwerveDriveOdometry odometry;
     private final Derivative acceleration = new Derivative();
     private final SwerveModulePosition[] modulePositions = new SwerveModulePosition[4];
     private final SwerveDriveInputsAutoLogged loggerInputs = new SwerveDriveInputsAutoLogged();
-    private final SwerveModuleState[] currentModuleStates = new SwerveModuleState[4];
 
-    private SwerveDrive(boolean isReal) {
+    private SwerveDrive(boolean isReal,
+                        int[] driveIds,
+                        int[] angleIds,
+                        int[] encoderIds) {
         if (isReal) {
```

Comment:
Subsystems shouldn't be aware of their "realness".

Reply:
We'll take care of that once it's merged into the 2024-common branch